backend/db-setup/README.md

```diff
@@ -0,0 +1,50 @@
+# Development Database Setup/Management Tools
+
+This directory contains a set of shell scripts intended to let you easily
+handle common development situations related to the database.
+
+## Configuration and management
+
+All of the scripts in this directory respect two environment variables:
+
+- `SR_DB_PORT` is the port on which the database is listening (default: 5432)
+- `SR_DB_HOST` is the host on which the database is running (default: localhost)
+
+This allows these scripts to be used for maintaining both the development and
+test databases (which generally run locally on different ports). It also allows
+them to be used against remote databases in deployed environments: please use
+care when attempting such maneuvers, as there is no warranty attached to this
+code.
+
+All of the scripts in this directory otherwise assume that you are using the
+default setup (i.e. database, user, and schema names) configured for dockerized
+databases in this directory, in the `dev` bean profile and the test application
+configuration.
+
+## Clear your database and start over
+
+In the event that you want to delete everything releated to SimpleReport,
+you can run `db-setup/nuke-db.sh` and it will do that. This deletes the schema
+that we use for our data, and is not reversible other than by restoring from a
+backup.
+
+To do this on your test database, just run './gradlew testDbReset` and gradle
```

Comment:
`'./gradlew testDbReset`` starting quote should be starting code tick

Reply:
(Also, thanks for the detailed documentation!)

Reply:
Good eye!